src/simulation/monsters/bosses/Zulrah.ts

```diff
@@ -72,7 +72,7 @@ const ZulrahTable = new LootTable()
 const TotalZulrahyTable = new LootTable()
 	.every(ZulrahTable)
```

Comment:
Instead of repeating the .every twice you can in the first every command say how many times it should be executed. .every(table, 2)

Reply:
Ah, good to know.